Formula/kong.rb

```diff
@@ -1,22 +1,37 @@
+require "fileutils"
+
 class Kong < Formula
   desc "Open source Microservices and API Gateway"
   homepage "https://docs.konghq.com"
+  license "Apache-2.0"
 
-  KONG_OPENRESTY_VERSION = "1.21.4.1".freeze
-  KONG_VERSION = "3.1.1".freeze
+  KONG_VERSION = "3.2.1".freeze
 
   stable do
-    url "https://download.konghq.com/gateway-src/kong-#{KONG_VERSION}.tar.gz"
-    sha256 "849c6b6fb116f760b107e6aa52c5eb3b2a7b143281853e33d7249b2db87c87ad"
+    url "https://github.com/Kong/kong/archive/refs/tags/#{KONG_VERSION}.tar.gz"
+    sha256 "f1583cd7ae1c8e29daa6008b2ea493c432b918d0cf3faf918891eeb314ac1499"
   end
 
   head do
     url "https://github.com/Kong/kong.git", branch: "master"
   end
 
-  depends_on "libyaml"
-  depends_on "coreutils"
-  depends_on "kong/kong/openresty@#{KONG_OPENRESTY_VERSION}"
+  depends_on "automake" => :build
+  depends_on "bazelisk" => :build
+  depends_on "cmake" => :build
+  depends_on "coreutils" => :build
+  depends_on "curl" => :build
+  depends_on "git" => :build
+  depends_on "libyaml" => :build
+  depends_on "m4" => :build
+  depends_on "openjdk" => :build
+  depends_on "perl" => :build
+  depends_on "protobuf" => :build
+  depends_on "python" => :build
+  depends_on "rust" => :build
```

Comment:
```suggestion
```
we don't need explict dependency of rust here, the rust toolchain is managed by bazel (otherwise we can't pin the rust toolchain to what we want)